ui/apps/platform/src/Containers/NetworkGraph/external/ExternalToClusterSideBar.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { ReactElement } from 'react';
+import {
+    Divider,
+    Flex,
+    FlexItem,
+    Stack,
+    StackItem,
+    Text,
+    TextContent,
+    TextVariants,
+    Toolbar,
+    ToolbarContent,
+    ToolbarItem,
+} from '@patternfly/react-core';
+import { EdgeModel } from '@patternfly/react-topology';
+
+import { TableComposable, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table';
+import { getEdgesByNodeId, getNodeById } from '../utils/networkGraphUtils';
+import {
+    CIDRBlockNodeModel,
+    CustomNodeModel,
+    ExternalEntitiesNodeModel,
+    ExternalNodeModel,
+} from '../types/topology.type';
+
+import { CidrBlockIcon, ExternalEntitiesIcon } from '../common/NetworkGraphIcons';
+import EntityNameSearchInput from '../common/EntityNameSearchInput';
+
+type CidrBlockSideBarProps = {
+    id: string;
+    nodes: CustomNodeModel[];
+    edges: EdgeModel[];
+};
+
+const columnNames = {
+    entity: 'Entity',
+    address: 'Address',
+    activeTraffic: 'Active traffic',
+};
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+function ExternalToClusterSideBar({ id, nodes, edges }: CidrBlockSideBarProps): ReactElement {
+    // component state
+    const [entityNameFilter, setEntityNameFilter] = React.useState<string>('');
+
+    // derived data
+    const externalToClusterNode = getNodeById(nodes, id) as ExternalNodeModel;
+    const externalNodes = nodes.filter(
+        (node) => node.data.type === 'CIDR_BLOCK' || node.data.type === 'EXTERNAL_ENTITIES'
+    ) as (ExternalEntitiesNodeModel | CIDRBlockNodeModel)[];
+
+    return (
+        <Stack>
+            <StackItem>
+                <Flex direction={{ default: 'row' }} className="pf-u-p-md pf-u-mb-0">
+                    <FlexItem>
+                        <TextContent>
+                            <Text component={TextVariants.h1} className="pf-u-font-size-xl">
+                                {externalToClusterNode?.label}
+                            </Text>
+                        </TextContent>
+                        <TextContent>
+                            <Text
+                                component={TextVariants.h2}
+                                className="pf-u-font-size-sm pf-u-color-200"
+                            >
```

Comment:
I think we already have an H1 on the page. Since this is a sub-section of the whole page, it would be more accessible to start this heading level with an H2, and so on. (The sizing seems custom regardless.)

```suggestion
                            <Text component={TextVariants.h2} className="pf-u-font-size-xl">
                                {externalToClusterNode?.label}
                            </Text>
                        </TextContent>
                        <TextContent>
                            <Text
                                component={TextVariants.h3}
                                className="pf-u-font-size-sm pf-u-color-200"
                            >
```